sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
Correct me if I'm wrong, but I think that's necessary on the plain print so that copy and paste will re-create the expression.